LICENSE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright JS Foundation and other contributors, https://js.foundation
```

Comment:
This as well as all other license files in the Appium project should be updated to apply to the OpenJS Foundation. Maybe something for another PR but would be nice to start this project right 😉

Reply:
@christian-bromann so the update would be `Copyright (c) OpenJS Foundation and other contributors`, right? 
cc @imurchie @jlipps

Reply:
That would be the minimum. Afaik there are no exact requirements to the license document so if this is a standard Apache-2 License you should be good to go.